### PR TITLE
Create registerCommands.ts

### LINKED ISSUE #1104: Register commands in a separate registerCommands.ts file instead of in extension.ts

### PR DESCRIPTION
**Pull request by alexweininger**
Fixes #1104

### CODE REVIEW

File: src/commands/registerCommands.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    registerBlobContainerGroupActionHandlers();
+    registerFileActionHandlers();
+    registerDirectoryActionHandlers();
+    registerFileShareActionHandlers();
+    registerFileShareGroupActionHandlers();
+    registerQueueActionHandlers();
+    registerQueueGroupActionHandlers();
+    registerStorageAccountActionHandlers();
+    registerTableActionHandlers();
+    registerTableGroupActionHandlers();
```
Comment (alexweininger) [inline src/commands/registerCommands.ts:51]: Going to just leave these for now. We can refactor these in a new PR later if we want.

**Comment by alexweininger** [comment]
Seems like this really screwed something up. Maybe a circular dependency?

**Review by bwateratmsft — DISMISSED** [review]

**Comment by nturinski** [comment]
Likely because of [this:](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=50635&view=logs&j=a5e52b91-c83f-5429-4a68-c246fc63a4f7&t=8932e18b-f2bd-5988-6524-8b3f226f2943&l=181) 

> ** TELEMETRY("vscode-azurestorage/activate", 0.14.3-alpha.2) properties={"isActivationEvent":"true","lastStep":"","result":"Failed","stack":"Object.<anonymous> extension.bundle.js:2:3268973\nfulfilled extension.bundle.js:2:3267422","error":"Error","errorMessage":"Could not find the Azure Resource Groups extension"}, measures={"duration":0.022,"mainFileLoad":0}

Easiest fix right now would be to move `registerCommands` above the rgApiProvider call.

**Comment by alexweininger** [comment]
> Likely because of [this:](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=50635&view=logs&j=a5e52b91-c83f-5429-4a68-c246fc63a4f7&t=8932e18b-f2bd-5988-6524-8b3f226f2943&l=181)
> 
> > ** TELEMETRY("vscode-azurestorage/activate", 0.14.3-alpha.2) properties={"isActivationEvent":"true","lastStep":"","result":"Failed","stack":"Object. extension.bundle.js:2:3268973\nfulfilled extension.bundle.js:2:3267422","error":"Error","errorMessage":"Could not find the Azure Resource Groups extension"}, measures={"duration":0.022,"mainFileLoad":0}
> 
> Easiest fix right now would be to move `registerCommands` above the rgApiProvider call.

Still didn't work 🤔

**Comment by nturinski** [comment]
Weird, worked for me https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=50700&view=results

**Comment by nturinski** [comment]
Did you move this line? https://github.com/microsoft/vscode-azurestorage/pull/1115/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626R55

**Comment by alexweininger** [comment]
> Did you move this line? [#1115 (files)](https://github.com/microsoft/vscode-azurestorage/pull/1115/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626R55)

Yeah, still getting this error on my machine
image

**Comment by alexweininger** [comment]
Found the circular dependency! Fixed it, this is ready for review now. 😄

File: src/AttachedAccountRoot.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (alexweininger) [inline src/AttachedAccountRoot.ts:1]: Had to put this in a separate file to fix the circular dependency

**Review by nturinski — APPROVED** [review]
Oh, did not realize that you weren't talking about the pipelines error. Good find!

**Pull request merged**